dev/idm-am-common-user/development/skaffold.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: skaffold/v1alpha5
+kind: Config
+build:
+  artifacts:
+  - image: gcr.io/engineering-devops/skaffold/oauth2/amster
+    context: ../amster
+  - image: gcr.io/engineering-devops/skaffold/oauth2/idm
+    context: ../idm
+  tagPolicy:
+    sha256: {}
+  local:
+    push: false
+deploy:
+  helm:
+    releases:
+    - name: sk-frconfig
+      chartPath: helm/frconfig
+      setValues:
+        domain: .forgeops.com
+        platform.idmProtectedByIg: true
```

Comment:
You will probably want to align with this PR too: https://github.com/ForgeRock/forgeops-init/pull/151